public/docs/ts/latest/guide/architecture.jade

```diff
@@ -92,21 +92,19 @@ figure
 
 .l-sub-section
   :marked
-    The `export` of `AppComponent` is just for show.
+    The `export` of `AppComponent` is just for demonstrative purposes in this example.
```

Comment:
It hurts to add so many big words here, although I agree "for show" is unclear. Maybe:

----
The `export` of `AppComponent` is just to show how to export; it isn't actually necessary in this example. A root module has no reason to *export* anything because other components don't need to *import* the root module.
----

If you think that's unclear, we could possibly add more words (though you know I'm generally against more words unless they're absolutely necessary!):

However, if another module *did* import the root module, then that module could use `AppComponent` in its component templates.